drivers/SmartThings/matter-window-covering/src/init.lua

```diff
@@ -276,6 +225,9 @@ local matter_driver_template = {
     capabilities.windowShadePreset,
     capabilities.battery,
   },
+  sub_drivers = {
+    subWindowCovering
```

Comment:
```suggestion
    -- for devices sending multiple positional updates on update
    require("matter-window-covering-position-updates-while-moving")
```
The previous name was unclear, which I think will cause confusion in the future. This is a little wordy, but the case is wordy as well. If anyone can think of a clearer name feel free to throw it out!

Also, be sure to change the name of the driver to follow this new name

Reply:
I think it's a clear name.